grails-app/services/org/pih/warehouse/data/LocationDataService.groovy

```diff
@@ -18,6 +18,7 @@ import org.pih.warehouse.core.Organization
 import org.pih.warehouse.importer.ImportDataCommand
 
 import javax.annotation.Nullable
+import javax.xml.bind.ValidationException
```

Comment:
wrong validation exception. should be grails validation exception.

Reply:
Removed, since it is not used.